doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -5,28 +5,65 @@
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Measurement Criteria](#3.2)
-  * [3.2.1 Storage and IOPS](#3.2.1)
-* [3.3 Measurement Types](#3.3)
-  * [3.3.1 Performance Measurements](#3.3.1)
-  * [3.3.2 Resiliency Measurements](#3.3.2)
-* [3.4 NFVI Test Cases](#3.4)
+* [3.2 Assumptions](#3.2)
+* [3.3 Requirement Type](#3.3)
+* [3.4 Profile Catalog](#3.4)
+* [3.5 Software & Hardware Reference](#3.5)
+* [3.6 Options & Extensions](#3.6)
+* [3.7 Measurement Criteria](#3.7)
+  * [3.7.1 Storage and IOPS](#3.7.1)
+* [3.8 Measurement Types](#3.8)
+  * [3.8.1 Performance Measurements](#3.8.1)
+  * [3.8.2 Resiliency Measurements](#3.8.2)
+* [3.9 NFVI Test Cases](#3.9)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="3.2"></a>
-## 3.2 Measurement Criteria
+## 3.2 Assumptions
+
+Content to be written:
+- Assumptions:  Automatable, Integrated with CICD tool chain
+
+<a name="3.3"></a>
+## 3.3 Requirement Type
+
+Content to be written:
+-  Type of requirement:  Bare metal, API, etc
+
+<a name="3.4"></a>
+## 3.4 Profile Catalog
+
+Content to be written:
+-  Table showing Profile Catalog
+
+<a name="3.5"></a>
+## 3.5 Software & Hardware Reference
+
+Content to be written:
+- Identify SW Reference
```

Comment:
is this refering to the software versions being part of the LFN compliance program? E.g., covering OpenStack Train to Ocata?

Reply:
@gkunz - this is in reference to a Specific SW Reference Profile, not the software version.  Similar to the Options and Extensions mentioned earlier, a clear description of the NFVI software profile to be evaluated under test from the RM-1/RA-1 requirements.

Reply:
So, the intention is to describe the NFVI reference profile which is subject to compliance testing. But why is this not the RA itself? The RA is the abstraction level above the RI, i.e. before picking a concrete implementation.

Reply:
This is the explicit profile.  We can refer back to RA for detailed info, but this section should describe and show what profile is being implemented.